<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saves</class>
 <widget class="QWidget" name="saves">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Saves</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/pic/savesbgr);</string>
  </property>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>93</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/back);
background: transparent</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>361</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DFPNewChuan-B5</family>
     <pointsize>52</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string>選擇存檔</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Slot_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>280</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/unbtn);
background: transparent</string>
   </property>
   <property name="text">
    <string>Empty Slot 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Slot_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>410</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/unbtn);
background: transparent</string>
   </property>
   <property name="text">
    <string>Empty Slot 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Slot_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>530</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/unbtn);
background: transparent</string>
   </property>
   <property name="text">
    <string>Empty Slot 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Slot_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>150</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/unbtn);
background: transparent</string>
   </property>
   <property name="text">
    <string>Empty Slot 1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
